plan/plan.go

```diff
@@ -258,18 +256,6 @@ func (p *baseLogicalPlan) buildKeyInfo() {
 	case *LogicalMaxOneRow, *LogicalExists:
 		p.maxOneRow = true
 	}
-	if len(p.children) == 1 {
-		switch p.self.(type) {
-		case *LogicalExists, *LogicalAggregation, *LogicalProjection:
```

Comment:
Is this logic handled in `build_key_info` too? If not, why remove this logic?

Reply:
Now `schema` is only handled by some particular plans, so we don't need initialize keys for every plan.